src/mixins/filing-template-mixin.ts

```diff
@@ -57,6 +57,9 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
   @Getter isEntityTypeSP!: boolean
   @Getter isEntityTypeGP!: boolean
   @Getter getCorrectedFilingType!: FilingTypes
+  @Getter getCorrectionErrorType!: CorrectionErrorTypes
```

Comment:
`getCorrectionErrorType` is [here](https://github.com/bcgov/business-edit-ui/blob/ebf983a0fd806f1d709c267e5ca866553e541735/src/store/getters/state-getters.ts#L162), but the correction error type is [here](https://github.com/bcgov/business-edit-ui/blob/ebf983a0fd806f1d709c267e5ca866553e541735/src/store/getters/state-getters.ts#L167).

Reply:
The `getCorrectionErrorType` getter is needed here because the correction error type is saved in a correction filing.